### PR TITLE
{tools}[foss/2019a] MoviePy 1.0.1 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by branfosj**
Not `multi-deps` as the Python 2 version requires a different versions of some libraries and extra ones compared to Python 3.

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/24617d8d2dded45213e8a36c9209ecc2 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ea7a3e92425f858d39e494007aef7a65 for a full test report.

File: easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.1-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('imageio', '2.6.1', {
+        'checksums': ['f44eb231b9df485874f2ffd22dfd0c3c711e7de076516b9374edea5c65bc67ae'],
+    }),
+    ('imageio-ffmpeg', '0.3.0', {
```
Comment (Micket) [inline easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.1-foss-2019a-Python-3.7.2.eb]: I'm a bit surprised that FFmpeg isn't a dependency for this package?

Comment (branfosj) [reply easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.1-foss-2019a-Python-3.7.2.eb]: Oops. On testing I get
```
RuntimeError: No ffmpeg exe could be found. Install ffmpeg on your system, or set the IMAGEIO_FFMPEG_EXE environment variable.
```

It seems that this does not get checked during the install - only at runtime. The user we installed this for is using it in conjunction with PyTorch and that has a FFmpeg dependency.

I'll get it fixed.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/551705f4dc18f895cd368aaf65454c6c for a full test report.

File: easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.1-foss-2019a-Python-3.7.2.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.1-foss-2019a-Python-3.7.2.eb]: With FFmpeg added as a dep:
```
$ python
Python 3.7.2 (default, Aug 28 2019, 11:29:43) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import imageio_ffmpeg
>>> imageio_ffmpeg.get_ffmpeg_exe()
'ffmpeg'
```

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/1174c86685e9a363c633b0c9e23a3c04 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/bb866393e2ed7f37e5ef014607dc402e for a full test report.

**Comment by Micket** [comment]
Going in, thanks @branfosj!

**Pull request merged**